test-projects/split/split_proj/models/metrics/metrics.yml

```diff
@@ -1,60 +0,0 @@
-version: 2
-
-metrics:
-  - name: revenue
-    label: Revenue
-    model: ref('orders')
-    description: "Income from all orders less tax"
-
-    calculation_method: sum
-    expression: order_total - tax_paid
-
-    timestamp: ordered_at
-    time_grains: [day, week, month, quarter, year]
```

Comment:
This is regrettable. Perhaps there's an opportunity to add back in metric tests / semantic model tests in 0.4.x 🤷🏻